File: features/offices/src/main/java/com/nivasafinance/features/offices/dto/OfficeResponse.java
package com.nivasafinance.features.offices.dto;

import com.nivasafinance.common.dto.AddressData;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class OfficeResponse {
    private Long id;
    private String name;
    private String key;
    private String code;
    private AddressData address;
    private Long parentId;
    private Boolean isActive;
}

File: features/offices/src/main/java/com/nivasafinance/features/offices/dto/OfficeResponse.kt
package com.nivasafinance.features.offices.dto

import com.nivasafinance.features.address.dto.AddressResponse
import java.util.UUID

data class OfficeResponse(
    val id: UUID,
    val name: String,
    val key: String,
    val code: String,
    val address: AddressResponse?
)
